### ISSUE TITLE
Using three-character delimiters.

### ISSUE THREAD

**Issue by danielfaust** [djc/askama#763]
I have an issue with trying to use the following in `askama.toml`

```
[[syntax]]
name = "python"
expr_start = "{@@"
expr_end = "@@}"
```

where I then get the compile error

```
error: length of delimiters must be two
```

This is due to the constraint set in [`config.rs`](https://github.com/djc/askama/blob/521964e12e7d4f0381c7d8d2c5ca1911c12a5466/askama_derive/src/config.rs#L167).

I just started with Rust and coming from Python I've used Jinja2 extensively, where I always used the following configuration

```
aiohttp_jinja2.setup(app, loader=jinja2.FileSystemLoader('html'), variable_start_string='{@@', variable_end_string='@@}')
```

I started with this three-character delimiter very early on and have years of templates using this format because this would ensure that it would never collide with JavaScript frameworks.

Is there a chance of allowing longer delimiters or is there a special reasoning for imposing this limitation?

Thank you a lot for putting your time into creating *askama*.

**Comment by djc** [comment]
I think the parser was originally optimized for two-character delimiters, but looking at the code now I no longer see code where this would be an issue. So maybe we can lift that restriction? @Kijewski do you remember this?

**Comment by Kijewski** [comment]
@djc, the need for two character delimiters was lifted in [#541](https://github.com/djc/askama/pull/541/files#diff-t48297b9b3a3f70c8521bedbaa0601f9e2b405eaa7a99fbffe0d9131e3207e478L202-L243). Yeah, I think its reasonable to remove the check in [config.rs:167](https://github.com/djc/askama/blob/6f52d0eef5a3178e1ebf7d4c7e2cf397756f57eb/askama_derive/src/config.rs#L167).

**Comment by djc** [comment]
Ah, cool, I couldn't quickly find it. @danielfaust would you be able to submit a PR to remove the restriction?

**Comment by danielfaust** [comment]
@djc, sure, I will do it. I will first review it a bit, but have one question to make it easier for me: Is it only the length of the corresponding block of code which needs to be adjusted? I would replace all instances of `.len() != 2` with `.len() < 2` and adapt the error message.

**Comment by djc** [comment]
> I would replace all instances of `.len() != 2` with `.len() < 2` and adapt the error message.

This sounds good to me!

**Comment by Kijewski** [comment]
Resolved in #820.
